bigtop-packages/src/common/giraph/install_giraph.sh

```diff
@@ -134,6 +135,7 @@ for i in giraph ; do
 # Workaround for GIRAPH-199
 export HADOOP_HOME=\${HADOOP_HOME:-/usr/lib/hadoop}
 export HADOOP_CONF_DIR=\${HADOOP_CONF_DIR:-/etc/hadoop/conf}
+export HADOOP_CLASSPATH=/etc/giraph/conf
```

Comment:
Shall this be something like 
HADOOP_CLASSPATH=${HADOOP_CLASSPATH}:/etc/giraph/conf
OR
HADOOP_CLASSPATH="`hadoop classpath`:/etc/giraph/conf"